bin/kona-exex/src/kona.rs

```diff
@@ -24,8 +25,35 @@ impl<Node: FullNodeComponents> KonaExEx<Node> {
         Self { ctx, cfg }
     }
 
+    /// Wait for the L2 genesis L1 block (aka "origin block") to be available in the L1 chain.
+    async fn wait_for_l2_genesis_l1_block(&mut self) -> Result<()> {
+        loop {
+            if let Some(notification) = self.ctx.notifications.recv().await {
+                if let Some(committed_chain) = notification.committed_chain() {
+                    let tip = committed_chain.tip().block.header().number;
+                    // TODO: commit the chain to a local buffered provider
+                    // self.chain_provider.commit_chain(committed_chain);
+
+                    if tip >= self.cfg.genesis.l1.number {
+                        break Ok(());
```

Comment:
you do not send `ExExEvent::FinishedHeight` in this case

Reply:
Good catch! In practice the main loop will also update the FinishedHeight on each new notification 
after the genesis block has been reached, which should be ok, but it's better to add it here too